crates/matrix-sdk/src/widget/machine/driver_req.rs

```diff
@@ -245,3 +251,33 @@ impl FromMatrixDriverResponse for SendEventResponse {
         }
     }
 }
+
+/// Ask the client to send a UpdateDelayedEventRequest with the given `delay_id`
+/// and `action`.
```

Comment:
Maybe mention the MSC here too
```suggestion
/// Ask the client to send a UpdateDelayedEventRequest with the given `delay_id`
/// and `action`. Defined by [MSC4157](https://github.com/matrix-org/matrix-spec-proposals/pull/4157)
```